ocs_ci/utility/framework/fusion_fdf_init.py

```diff
@@ -104,6 +106,9 @@ def init_cli(self, args: list) -> list:
             default=[],
             help="Path to config file. Repeatable.",
         )
+        parser.add_argument(
+            "--report", default=None, help="Filepath for generated junit report"
```

Comment:
The `report` name seems to be quite generic and not directly clear what kind of report it provides (e.g. xml, html,...). In `ocs-ci` (or better in pytest) there is `--junit-xml`  parameter for that, what about using the same?